systemtest/020-copy.bats

```diff
@@ -14,7 +14,7 @@ function setup() {
 # From remote, to dir1, to local, to dir2;
 # compare dir1 and dir2, expect no changes
 @test "copy: dir, round trip" {
-    local remote_image=docker://busybox:latest
+    local remote_image=docker://docker.io/library/busybox:latest
```

Comment:
(None of the `docker://` edits should be necessary: in the `docker://` syntax, short names always mean `docker.io`. But they don’t hurt at all and it’s nice to get in the habit.

Or is there a case where the `docker://` syntax triggers a search? I’d very much like to know about it, and get rid of it.)

Reply:
Thank you! This ("short names always mean docker.io") explains why I couldn't get the tests to fail using a doctored `registries.conf` without docker.io! I spent a long time wondering, then finally decided to go with FQINs anyway. 

So: no, don't worry, there is no case (that I've seen) where `docker://` triggers a registry search. I was just trying to get the tests using full names consistently. Thanks again!